### DISCUSSION TITLE
Audi MQB is a little bit different, sadly -_-

### DISCUSSION THREAD

**Discussion by ivelliosy** [jilleb/MQB-FPA#24] [Q&A]
Hi Jilleb and team. I successfully edited my original DS in 010 editor, and successfully keep DSG not resetting. Also added economy mode to my invididual channels. However, I am not able to make drive profile names shown in cluster. It just doesn't show anything! Original DS was also like it, but I tried Golf and Octavia datasets, they are showing names on cluster. Maybe there are some more bytes regarding aid banner? Because I edited the file well acc. to dataset and template in wiki.

One last problem, everytime i switch on and off the car, profile goes back to invidual, no matter in which profile it was in.

Thanks!
[19_Audi_A3_3Q0907530AC_Original & FPA Edited.zip](https://github.com/jilleb/MQB-FPA/files/6015002/19_Audi_A3_3Q0907530AC_Original.FPA.Edited.zip)

**Comment by batal131** [comment]
Hey. Just checked your profile and it's very strange :)

I have factory dataset with no profile memory (339h) - car always returns no "Normal" profile.
I've changed it (339h-33Ch) to save profiles. It works for me! Everything except DSG...

I've checked your files and I see that you've changed the same. But memory for you works only for DSG...

**Comment by Mich795** [comment]
Hi, I also edited the DS for A3 8V 3Q0907530AC Gateway to memorize the selection of the efficency and individual profile. i've the same problem "everytime i switch on and off the car, profile goes back to invidual, no matter in which profile it was in". Also for me the selection of the driving profile before turning off the car is not memorized. 
Does someone solved it?
